site/layouts/partials/navbar.html

```diff
@@ -1,10 +1,9 @@
-<div id="navbar" class="black">
+<div id="navbar" class="docker-blue">
   <span id="sidebar-toggle" onclick="toggleSidebar()">&#9776;</span><span id="navbar-title"><a href="{{.Site.BaseURL}}">kind</a></span><span id="github"><a href="https://github.com/kubernetes-sigs/kind/"><img alt="github" title="github" src="{{ "third_party/GitHub-Mark-120px-plus.png" | relURL }}"></a></span>
 </div>
-<div id="noticebar" class="black">
-  <div><h2>Black lives matter.</h2></div>
-  <p>We stand in solidarity with the Black community.</p>
-  <p>Racism is unacceptable.</p>
-  <p>It conflicts with <a href="https://git.k8s.io/community/values.md">the core values of the Kubernetes project</a> and our community does not tolerate it.</p>
-  </p>
+<div id="noticebar" class="docker-blue">
+  <h2>Dockershim <a href="https://github.com/kubernetes/kubernetes/pull/94624">deprecation</a> does NOT impact kind.&nbsp;&nbsp;🐳</h2>
```

Comment:
https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md#deprecation seems like it might be better

Reply:
done.